x/dex/keeper/expected_keeper.go

```diff
@@ -54,6 +54,8 @@ type IKeeper interface {
 	CompleteWithdraw(ctx sdk.Context, addr sdk.AccAddress) error
 	IterateWithdrawInfo(ctx sdk.Context, fn func(index int64, withdrawInfo types.WithdrawInfo) (stop bool))
 	DeleteWithdrawCompleteTimeAddress(ctx sdk.Context, timestamp time.Time, delAddr sdk.AccAddress)
+	GetTokenPairMaxID(ctx sdk.Context) (tokenPairMaxID uint64)
+	SetTokenPairMaxID(ctx sdk.Context, tokenPairMaxID uint64)
```

Comment:
I think MaxTokenPairID is better than TokenPairMaxID, and should keep naming consistent:
GetTokenPairMaxID-> GetMaxTokenPairID
SetTokenPairMaxID-> SetMaxTokenPairID